lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -9,6 +9,8 @@
  *
  * @license Mozilla Public License : http://www.mozilla.org/MPL/
  */
+use Lizmap\Project\ProjectMetadata;
```

Comment:
@laurentj I needed to add this `use` here which I did not need in master . Do you know why ?

Reply:
Because 
1. you created the file ProjectMetadata.php into the lizmap/lib/ directory, which is declared into the module.xml, as a directory which have classes having a namespace starting with `Lizmap\`, and so that are auto-loadable
2. in ProjectMetadata.php, you declared that the class is into the namespace `Lizmap\Project`

All is fine, you can keep as is.